maps/sccv_horizon/sccv_horizon-1_deck_1.dmm

```diff
@@ -16623,6 +16623,15 @@
 /obj/machinery/camera/network/mining{
 	c_tag = "Mining - Locker Room"
 	},
+/obj/structure/closet/secure_closet/guncabinet/exped,
+/obj/item/material/hatchet/machete,
+/obj/item/material/hatchet/machete,
+/obj/item/material/hatchet/machete,
+/obj/item/clothing/accessory/holster/utility/machete,
+/obj/item/clothing/accessory/holster/utility/machete,
+/obj/item/clothing/accessory/holster/utility/machete,
+/obj/item/gun/energy/laser/shotgun/research,
+/obj/item/gun/energy/laser/shotgun/research,
```

Comment:
map these out again since the locker is mapped in.

Reply:
yeah the item code didn't work when i tested it so i just mapped in the cabinet